### PR TITLE
add microkit annoncement text

### PR DESCRIPTION
**Pull request by bbrcknl**

### CODE REVIEW

File: content/news/2023.pml [hunk 1]
```diff
@@ -54,6 +54,19 @@ Older News: <%=reflist%>
...
 
+<!----------------------------------------------------------->
+<div class="news-item tag-link" id="microkit">
+  <a href="#microkit"><span class="tag-span"># </span></a>
+  25 October 2023: seL4 Microkit officially supported by the seL4 Foundation
+</div>
+<div>
+  <p>
+     The seL4 Microkit, formerly known as the Core Platform, is an operating system framework on top of seL4 provides a small set of simple abstractions that ease the design and implementation of statically structured systems on seL4, while still leveraging the kernel’s benefits of security and performance. The Microkit is distributed as an SDK that integrates with the developer’s build system of choice, significantly reducing the barrier to entry for new users of seL4.
+  </p>
```
Comment (Ivan-Velickovic) [inline content/news/2023.pml:65]: Do we want some extra context such as "The seL4 Microkit has been developed in collaboration between Breakaway Consulting and Trustworthy Systems and is now an official seL4 Foundation project, making it part of the seL4 eco-system."?

Comment (bbrcknl) [reply content/news/2023.pml:65]: @Ivan-Velickovic That's a good idea. I suggest adding the microkit logo (when done), TS logo and Breakaway logo too. I've pushed the news item date to 27 Oct.

File: content/news/2023.pml [hunk 2]
```diff
@@ -54,6 +54,31 @@ Older News: <%=reflist%>
...
+    <a href="https://brkawy.com/">
+      <img src="/Foundation/Membership/LOGOS/Brkawy.png" style="width: 200px; padding:20px; float: right" alt="Breakaway Consulting logo">
+    </a>
+    <div class="style:clear:both"></div>
+    <a href="https://unsw.edu.au/">
+    <img src="/Foundation/Membership/LOGOS/UNSW.svg" style="width: 200px; padding:20px; float: right" alt="UNSW logo">
+    </a>
+  </div>
+  <p>
+     The seL4 Microkit, formerly known as the Core Platform, is an operating system framework on top of seL4 provides a small set of simple abstractions that ease the design and implementation of statically structured systems on seL4, while still leveraging the kernel’s benefits of security and performance. The Microkit is distributed as an SDK that integrates with the developer’s build system of choice, significantly reducing the barrier to entry for new users of seL4.
```
Comment (june-andronick) [inline content/news/2023.pml:73]: We should really have a link here: https://docs.sel4.systems/projects/microkit/

Comment (june-andronick) [reply content/news/2023.pml:73]: Also: typo: "provides" -> "that provides"

**Review by june-andronick — COMMENTED** [review]
Thanks for starting this. Where is it up to? When will it be out?

File: content/news/2023.pml (hunk 2 above)
Comment (bbrcknl) [reply content/news/2023.pml:73]: @june-andronick Just waiting on the microkit logo. I've pencilled the release for 8 Nov.

**Review by Ivan-Velickovic — APPROVED** [review]

**Pull request merged**